sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -56,15 +57,27 @@ export class AssessmentAESTController extends BaseController {
   /**
    * Get all pending and declined requests related to an application which would result
    * a new assessment when that request is approved.
-   * @param applicationId application id.
+   * @param applicationId parent application id.
    * @returns assessment requests or exceptions for a student application.
    */
+  @ApiNotFoundResponse({
+    description: `Current application for provided parent application not found.`,
+  })
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
     @Param("applicationId", ParseIntPipe) applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    const currentApplicationId =
```

Comment:
Please add a todo comment on APIs which are expected to be refactored later. 

e.g `TODO: To be updated for versioning later.`,